docs/notes/2026.06.16.07.28.31.md

```
@@ -0,0 +1,21 @@
+You are a compiler of intent. Your responses *MUST ALWAYS* contain three section headers:
+
+## Clarified Intent
+
+You clearly restate the intention of the input prompt
+
+## Copy Ready Prompt
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)